### DISCUSSION TITLE
Unable to create/import dashboard with data source Infinity data source via HTTP API request

### DISCUSSION THREAD

**Discussion by max420max** [grafana/grafana-infinity-datasource#752] [Specific APIs]
Hello ,

I am trying to use Infinite data source to create/import a dashboard .

API NAME : /api/dashboards/db

Dashboard is created but i do not see infinity data source are created inside the new dashboard.
Even when i tried to import the JSON template with Infinity data source its give always bad request however same template is working fine via GUI.

Note, Infinity plugin already installed

example:
API name: /api/dashboards/db
/api/datasources
Body:

{
"dashboard": {
"id": null,
"uid": null,
"title": "Production Overview1",
"tags": [ "templated" ],
"timezone": "browser",
"schemaVersion": 16,
"refresh": "25s",

"panels": [
{
"id": 11,
"uid": "bbc41ab0-6a05-40c5-96a2-40fa81386063",
"orgId": 1,
"name": "Infinity-1",
"type": "yesoreyeram-infinity-datasource",
"typeLogoUrl": "public/plugins/yesoreyeram-infinity-datasource/img/icon.svg",
"access": "proxy",
"url": "",
"user": "",
"database": "",
"basicAuth": false,
"basicAuthUser": "",
"withCredentials": false,
"isDefault": false,
"jsonData": {},
"secureJsonFields": {},
"version": 1,
"readOnly": false

  }
],
"message": "Made changes to xyz",
"overwrite": true
}
}

Output:

Postman i can see 200 success but when i open i grafana GUI , i see dashbaord created and panel showing me below error

"Panel plugin not found: yesoreyeram-infinity-datasource"

Status: 500. Message: error getting data frame. error getting response from url https://raw.githubusercontent.com/grafana/grafana-infinity-datasource/main/testdata/users.json. no response received. Error: Get "https://raw.githubusercontent.com/grafana/grafana-infinity-datasource/main/testdata/users.json": dial tcp: lookup raw.githubusercontent.com on 10.43.0.10:53: no such host

curl -X POST -H “Content-Type: application/json” -u admin:xxxxx
-d @12344j11-1706621082586.json
http://xxxxxxxxxxxxx/api/dashboards/db

even curl command get success but i don' see any dashboard created in Grafana GUI

**Comment by zoltanbedi** [comment]
Please don't open multiple issues.
